samples/gtk/build.sh

```diff
@@ -3,18 +3,36 @@
 DIR=$(cd "$(dirname "${BASH_SOURCE[0]}" )" && pwd )
 PATH=$DIR/../../dist/bin:$DIR/../../bin:$PATH
 
-IPREFIX_macbook=-I/opt/local/include
-#IPREFIX_macbook=-I/usr/local/include
 IPREFIX_linux=-I/usr/include
 
-if [ x$TARGET == x ]; then
+if [ -z $TARGET ]; then
 case "$OSTYPE" in
   darwin*)  TARGET=macbook ;;
   linux*)   TARGET=linux ;;
   *)        echo "unknown: $OSTYPE" && exit 1;;
 esac
 fi
 
+
+while getopts ":I:" opt; do
+  case $opt in
+    I) IPREFIX="$OPTARG"
+    ;;
+  esac
+done
+
+if [ "$TARGET" == 'macbook' ] && [ ! -z $IPREFIX ]; then
+    IPREFIX_macbook=-I$IPREFIX
+elif [ "$TARGET" == 'macbook' ]; then
+    if [ -d /usr/local/include ]; then
+        IPREFIX_macbook=-I/usr/local/include
```

Comment:
This better be fixed by adding more -I and -L statements in compilerOpts.osx and linkerOpts.osx in src/main/c_interop/gtk3.def, to avoid excessive logic here.

Reply:
Can do that, but it will remove the options to manually add it. 
Also to support both `/usr` and `/opt` it will create a very long params.

    compilerOpts.osx = -I/usr/local/include/gtk-3.0 -I/usr/local/include/glib-2.0 -I/usr/local/lib/glib-2.0/include \
    -I/usr/local/include/pango-1.0 -I/usr/local/include/cairo -I/usr/local/include -I/usr/local/include/gdk-pixbuf-2.0 \
    -I/usr/local/include/atk-1.0 -I/opt/local/include/gtk-3.0 -I/opt/local/include/glib-2.0 \
    -I/opt/local/lib/glib-2.0/include -I/usr/local/include/pango-1.0 -I/opt/local/include/cairo -I/opt/local/include \
      -I/opt/local/include/gdk-pixbuf-2.0 -I/opt/local/include/atk-1.0

Other options is to include `/usr` in `compilerOpts.osx` and keep the cinterop command as-is.